src/containers/treePage/index.tsx

```diff
@@ -0,0 +1,248 @@
+  width: 90%;
+  margin: auto;
+`;
+
+const EntrySpace = styled.div`
+  display: flex;
+  flex-wrap: wrap;
+`;
+
+const TreeCareTitle = styled(Paragraph)`
+  margin: 15px 15px 50px;
+  font-size: 24px;
+  font-weight: bold;
+  color: ${ DARK_GREEN }
+`;
+
+const TreeCareContainer = styled.div`
+  border: solid 1px ${LIGHT_GREY};
+  height: 80%;
+  margin-top: 80px;
+  overflow: auto;
+`;
+
+const CareEntry = styled.div`
+  margin: 15px;
+`;
+
+const StewardshipContainer = styled.div`
+  margin-top: 40px;
+`;
+
+const EntryDate = styled(Paragraph)<TitleProps>`
+  display: inline;
+  text-align: center;
+  line-height: 0px;
+  font-size: 18px;
+  font-weight: bold;
+  color: ${ DARK_GREEN }
+`;
+
+const EntryMessage = styled(Paragraph)`
+  display: inline;
+  text-align: center;
+  line-height: 0px;
+  color: ${TEXT_GREY};
+`;
+
+const dummyCare: TreeCare[] = [
+  {
+    date: "March 28th",
+    message: "Was cleared of waste and weeded"
+  },
+  {
+    date: "March 27th",
+    message: "Was cleared of waste and weeded"
+  },
+  {
+    date: "March 26th",
+    message: "Was cleared of waste and weeded"
+  },
+  {
+    date: "March 25th",
+    message: "Was cleared of waste and weeded"
+  },
+  {
+    date: "March 24th",
+    message: "Was cleared of waste and weeded"
+  },
+  {
+    date: "March 23rd",
+    message: "Was cleared of waste and weeded"
+  },
+]
+
+const dummyTree: SiteProps = {
+  siteId: 1,
+  blockId: 1,
+  lat: 2,
+  lng: 2,
+  city: 'Boston',
+  zip: '02115',
+  address: '100 Super Tree Way',
+  entries: [{
+    id: 1,
+    username: 'Speak for the trees user',
+    updatedAt: 100000000,
+    treePresent: true,
+    status: 'good',
+    genus: 'biggus treebus',
+    species: 'larggeus',
+    commonName: 'large tree', 
+    confidence: 'good',
+    diameter: 1000000,
+    circumference: 3140000,
+    coverage: 'leaves',
+    discoloring: true,
+    pooling: false,
+    light: true,
+    bicycle: true,
+    fence: 'yes',
+    treeNotes: 'I think it looks pretty good',
+    siteNotes: 'dirty',
+  }]
+};
+
+const TreePage: React.FC<SiteProps> = ({
+  address,
+  entries
+}) => {
+
+  const latestEntry: SiteEntry = entries[entries.length - 1];
+
+  const description = "Dreamcatcher kogi taiyaki keytar. Swag typewriter craft beer cronut pok pok gentrify flannel salvia deep v pork belly"
+
+  return(
+    <>
+      <Helmet>
+        <title>Tree</title>
+        <meta
+          name="Individual Tree Page"
+          content="Description of a particular tree site, allows users to adopt a tree and monitor their stewardship activities."
+        />
+      </Helmet>
+      <PageLayout>
+        <TreePageContainer> 
+          {/*Change to tree map route once that page is finished*/}
+          <ReturnButton to={Routes.HOME}> 
+              {`<`} Return to Tree Map
+          </ReturnButton> 
+          <TreeMainContainer>
+          <Row>
+            <Col span={7}>
+              <TreeInfoContainer>
+                <TreeImage
```

Comment:
Can delete/comment this out for now or find a better universal placeholder - I don't think we'll be adding individual tree images for now